src/iperf3test_cli.py

```diff
@@ -0,0 +1,161 @@
+                                         usage='''iperf3test bandwidth <ip1> <ip2>''')
+        parser.add_argument("client", help="The client IP address.")
+        parser.add_argument("server", help="The server IP address.")
+        args = parser.parse_args(sys.argv[2:])
+
+        test = IPerf3Test(args.server, args.client)
+        test.run_bandwidth_test()
+
+
+class IPerf3Test:
+
+    def __init__(self, server_public_ip, client_public_ip):
+        inventory = load_inventory()
+
+        self.client = find_host_with_public_ip(inventory, client_public_ip)
+        if self.client is None:
+            exit_with_error(f"Could not find client [{client_public_ip} in the inventory]")
+
+        self.server = find_host_with_public_ip(inventory, server_public_ip)
+        if self.server is None:
+            exit_with_error(f"Could not find server [{server_public_ip} in the inventory]")
+
+    def run_bandwidth_test(self):
+        client_args = "--parallel 1 --time 20"
+        server_args = ""
+
+        server_thread = threading.Thread(target=start_server, args=(self.server, server_args))
+        server_thread.start()
+
+        client_thread = threading.Thread(target=start_client, args=(self.client, self.server, client_args))
+        client_thread.start()
+
+    def run_packet_count_test(self):
+        run_duration = 20
+        client_args = f"--parallel 128 --time {run_duration} --set-mss 89"
+        server_args = ""
+
+        print(f"Testing pps for {self.client['public_ip']} > {self.server['public_ip']}")
+
+        server_thread = threading.Thread(target=start_server, args=(self.server, server_args, True))
+        server_thread.start()
+
+        client_thread = threading.Thread(target=start_client, args=(self.client, self.server, client_args, True))
+        client_thread.start()
+
+        server_pps_thread = threading.Thread(target=start_pps, args=(self.server, run_duration))
+        server_pps_thread.start()
+
+        server_thread.join()
+        client_thread.join()
+        server_pps_thread.join()
+
+        print(f"Testing pps for {self.client['public_ip']} < {self.server['public_ip']}")
+
+        server_thread = threading.Thread(target=start_server, args=(self.client, server_args, True))
+        server_thread.start()
+
+        client_thread = threading.Thread(target=start_client, args=(self.server, self.client, client_args, True))
+        client_thread.start()
+
+        server_pps_thread = threading.Thread(target=start_pps, args=(self.server, run_duration))
+        server_pps_thread.start()
+
+
+def start_client(client, server, args, silent=False):
+    sleep(2)
+    ssh_client = Ssh(
+        client["public_ip"],
+        client["ssh_user"],
+        client["ssh_options"],
+        log_enabled=False,
+        use_control_socket=False)
+    ssh_client.connect()
+    server_ip = server["private_ip"]
+    iperf_cmd = f"iperf3 -c {server_ip} -p 3000 {args} --connect-timeout 30000"
```

Comment:
@pveentjer  the port was originally set to 5701 and you added the udp ingress rule for port 3000. So maybe that's the reason you had issues with running UDP?